Sources/PostgresNIO/New/PostgresQuery.swift

```diff
@@ -43,6 +43,7 @@ extension PostgresQuery {
             self.sql.append(contentsOf: literal)
         }
 
+        @_disfavoredOverload
```

Comment:
It's not satisfactory to me to need to use `@_disfavoredOverload` here, especially not twice. Is there another way to make the other overload more preferred? (Keep in mind that `@_disfavoredOverload` exists to work around constraint solver bugs, _not_ to let you change the language's overload resolution rules at a whim.)

Reply:
See https://github.com/vapor/postgres-nio/pull/410#discussion_r1334975466

Reply:
Outdated, see https://github.com/vapor/postgres-nio/pull/410#issuecomment-1732296628